packages/trpc/server/routers/viewer/availability.tsx

```diff
@@ -459,6 +467,11 @@ const getDefaultScheduleId = async (userId: number, prisma: PrismaClient) => {
     },
   });
 
+  await prisma.user.update({
```

Comment:
An update in a getter is very unexpected, function probably needs renaming (or logic modified)

Reply:
https://github.com/calcom/cal.com/pull/7789/commits/7b2881b1f487a24be560662cf0fe281c0a295f8f